website/docs/allmodulespublic.md

```diff
@@ -0,0 +1,26 @@
+allmodulespublic
+
+```lua
+allmodulespublic "value"
+```
+
+### Parameters ###
+
+`value` one of:
+* `On` - All C++ modules in the given project(s) will be public.
+* `Off` - Not all C++ modules in the given project(s) will be public.
```

Comment:
How to make modules public without that?

Reply:
Using "Project properties > VC++ Directories > Public C++ module directories", which is certainly implementable in Premake. But that's more involved, and should probably be done in a separate PR.